spec/features/user_recievs_email_after_microhood_is_reviewed_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'User is notified via email after review of their created microhood' do
+  scenario 'user reviews microhood' do
+    ActionMailer::Base.deliveries = []
+    user = FactoryGirl.create(:user)
```

Comment:
I would do a lot of this set up in let/ before blocks. It will make your code look cleaner and be more readable.